modules/fastq_filters.py

```diff
@@ -0,0 +1,99 @@
+def gc_content(seq: str) -> float:
+    """
+    Supporting function
+    Function counts GC-content in sequence, and returns result in %
+        Parameters:
+            seq (str): oligo- and polynucleotide  sequence
+        Returns:
+            (float): % of GC-content
+    """
+    n = 0
+    for nucl in seq:
+        if nucl == 'c' or nucl == 'g' or nucl == 'C' or nucl == 'G':
+            n += 1
+    return 100 * n / len(seq)
+
+
+def filter_gc(seqs: dict, gc_bounds_both_side=(0, 100)) -> dict:
+    """
+    This function selects sequences with the GC content of your interest
+    :parameters:
+        seqs: dictionary of FASTQ sequences {name: (sequence, quality)}
+        gc_bound: interval for the of acceptable GC content, in %
+    :return:(dict) new dictionary consists of selected sequences
+    """
+    d_new = {}

```

Comment:
Ну раз у тебя на входе seqs , мб тут как нибдуь типа
```suggestion
    selected_seqs = {}
```

А то d_new прям микро-загадки для читающих:)